documentation/URL_SPEC.md

```diff
@@ -0,0 +1,159 @@
+| `iso`    | Whether the isosurface is enabled.                                                        | `1` (enabled) or `0` (disabled)                                                                | `0`                                    | `?c0=iso:1`                         |
+| `isv`    | Isosurface value, the intensity value at which to generate the isosurface.                | Number in the range `[0, 255]`                                                                 | `128`                                  | `?c0=isv:195`                       |
+| `col`    | Base channel color, applied to volumes and isosurfaces.                                   | 6-digit hex color                                                                              | (varies by index)                      | `?c0=col:af38c0`                    |
+| `clz`    | Colorize, used for segmentations where each ID should be a different color.               | `1` (enabled) or `0` (disabled)                                                                | `0`                                    | `?c0=clz:1`                         |
+| `cza`    | Colorize alpha, the opacity of the colorize effect.                                       | Number in the range `[0, 1]`                                                                   | `1.0`                                  | `?c0=cza:1.0`                       |
+| `isa`    | Isosurface alpha, the opacity of the isosurface.                                          | Number in the range `[0, 1]`                                                                   | `1.0`                                  | `?c0=isa:1.0`                       |
+| `cpe`    | Use and show the control points instead of the ramp on load.                              | `1` (enabled) or `0` (disabled)                                                                | `0`                                    | `?c0=cpe:1`                         |
+| `cps`    | Control points for the transfer function. If provided, overrides the `lut` field.         | List of `bin_index:opacity:color` triplets, separated by a colon ([_see 'Binning'_](#binning)) | `0:0:ffffff:255:1:ffffff`              | `?c0=cps:0:0:ff0000:150:0.5:ffff00` |
+| `rmp`    | Raw ramp values. If provided, overrides the `lut` field when calculating the ramp values. | Two float bin indices, formatted as `min:max` ([_see 'Binning'_](#binning))                    | `0:255`                                | `?c0=rmp:0:255`                     |
+| `lut`    | Lookup table (LUT) to map from volume intensity values to opacity.                        | [_See 'Lookup Table'_](#lookup-table-lut)                                                      | `0:255`                                | `?c0=lut:0:255`                     |
```

Comment:
do lut and rmp map the same values - a min and a max?  I think the only difference is that lut accepts the shorthand / prefixed values?

Reply:
Yes! There is some different behavior though, since setting `lut` will set both the control points and the ramp, unless one or both are overridden.

Reply:
When we send share links from VV it's always with `rmp` and `cps` and we don't include `lut`.

Reply:
Maybe we should remove lut then!

Reply:
I think it may still be a good idea to keep it? This way a user can still say, "load any volume with autoij"